wic/omnect-os.phytec.wks.in

```diff
@@ -1,7 +1,7 @@
 part U-BOOT --no-table --offset ${BOOTLOADER_SEEK} --source rawcopy --sourceparams="file=bootloader.versioned.bin"
-part --fixed-size ${OMNECT_PART_SIZE_UBOOT_ENV}K --no-table --offset ${OMNECT_PART_OFFSET_UBOOT_ENV1} --source rawcopy --sourceparams="file=omnect_uboot_env_ff.img"
-part --fixed-size ${OMNECT_PART_SIZE_UBOOT_ENV}K --no-table --offset ${OMNECT_PART_OFFSET_UBOOT_ENV2} --source rawcopy --sourceparams="file=omnect_uboot_env_ff.img"
-part --fixed-size ${OMNECT_PART_SIZE_BOOT}K      --fstype=vfat --align 4096 --label boot   --source bootimg-partition --active
+part --no-table --offset ${OMNECT_PART_OFFSET_UBOOT_ENV1} --source rawcopy --sourceparams="file=uboot-env.bin"
+part --no-table --offset ${OMNECT_PART_OFFSET_UBOOT_ENV2} --source rawcopy --sourceparams="file=uboot-env.bin"
+part --offset ${OMNECT_PART_OFFSET_BOOT} --fixed-size ${OMNECT_PART_SIZE_BOOT}K --fstype=vfat --label boot --source bootimg-partition --active
```

Comment:
I know that didn't change with that PR, but for clarity it might be better use 'K' suffix also for offset values.

Reply:
done